client/js/compute.js

```diff
@@ -684,6 +684,48 @@ const compute_ops = [
     sample: 'var a = randRange ${x} ${y} ${z}',
     call: function(v, x, y, z) { return v = Math.round(Math.floor((Math.random() * (y - x) / (z || 1))) * (z || 1) + x) },
     hash: '4758cba8f404cfe3ef0a300cebf3938a'
+  },
+  {
+    name: 'colorContrast',
+    desc: 'converts color in any format that the browser can interpret to another color in the same hue but with different luminance',
+    sample: 'var a = colorContrast ${x} ${y}',
+    call: function(v, x, y) { return v = contrastAnyColor(x, y); }, 
+    hash: '83181e89c7f0245a49644ce864625481'
+  },
+  {
+    name: 'colorToHex',
+    desc: 'converts color in any format that the browser can interpret to hex',
+    sample: 'var a = colorToHex ${x}',
+    call: function(v, x) { return v = toHex(x); },
+    hash: '2eff46ec1af852a29fa4c14dcf685c00'
+  },
+  {
+    name: 'colorToRGB',
+    desc: 'converts color in any format that the browser can interpret to RGB in format rgb(0,9,210)',
+    sample: 'var a = colorToRGB ${x}',
+    call: function(v, x) { return v = toRGB(x); },
+    hash: 'faa247cc277b7e9ad645e55a51e47195'
+  },
+  {
+    name: 'colorContrastRatio',
+    desc: 'compares 2 colors in any format that the broswer can interpret to obtain the contrast ratio in range 1-21',
+    sample: 'var a = colorContrastRatio ${x} ${y}',
+    call: function(v, x, y) { return v = calcContrast(x, y); },
+    hash: 'f0c87b733f49b7af419698cdf6ed1137'
+  },
+  {
+    name: 'colorLuminance',
+    desc: 'accepts color in any format the browser can interpret and returns the luminance value',
+    sample: 'var a = colorCLuminance ${x}',
+    call: function(v, x, y) { return v = calcLuminance(x); },
+    hash: 'f4284956510e3fe59c2babed665b544d'
+  },
+  {
+    name: 'colorCreateHue',
+    desc: 'returns a random, as visually distinct as possible, hex color using linear interpolation',
+    sample: 'var a = colorCreateHue',
```

Comment:
Visually distinct from what? The example doesn't use a parameter but the function seems to support one.